core-web/libs/edit-content/src/lib/directives/tab-view-insert/tab-view-insert.directive.ts

```diff
@@ -0,0 +1,77 @@
+import {
+    AfterViewInit,
+    Directive,
+    inject,
+    input,
+    Renderer2,
+    TemplateRef,
+    ViewContainerRef
+} from '@angular/core';
+
+import { TabView } from 'primeng/tabview';
+
+@Directive({
+    selector: '[dotTabViewAppend]',
+    standalone: true
+})
+export class TabViewInsertDirective implements AfterViewInit {
+    $prependTpl = input<TemplateRef<unknown> | null>(null, { alias: 'dotTabViewPrepend' });
+    $appendTpl = input<TemplateRef<unknown>>(null, { alias: 'dotTabViewAppend' });
```

Comment:
$appendTpl should be typed as input<TemplateRef<unknown> | null> to accept null as the initial value.

Reply:
added to both inputs.